lib/APPReinforcementLearning/src/ReadyState.h

```diff
@@ -117,7 +117,10 @@ class ReadyState : public IState
     };
 
     /** Duration of the selected mode in ms. This is the maximum time to select a mode. */
-    static const uint32_t mode_selected_period = 200U;
+    static const uint32_t MODE_SELECTED_PERIOD = 100U;
```

Comment:
Now it might work with the timeout value, but I don't think you can change between the modes by the buttons. How shall a human press a button within 150ms before the transitition will take place? Please check this manually!

BTW why does the getSelectedMode() returns uint8_t instead of Mode?

Reply:
The getSelectedMode() function originally returned uint8_t instead of the mode enum because this was a bug in the implementation. I fixed this bug by changing the return value to the mode enum.